docs/document/content/user-manual/shardingsphere-proxy/cdc/usage.cn.md

```diff
@@ -0,0 +1,269 @@
+            // 开始 CDC 数据同步，返回的 streamingId 是这次 CDC 任务的唯一标识，CDC Server 生成唯一标识的依据是 订阅的数据库名称 + 订阅的表 + 是否是全量同步
+            String streamingId = cdcClient.startStreaming(new StartStreamingParameter("sharding_db", Collections.singleton(SchemaTable.newBuilder().setTable("t_order").build()), true));
+            log.info("Streaming id={}", streamingId);
+            // 防止 main 主线程退出
+            cdcClient.await();
+        }
+    }
+}
+```
+
+主要有4个步骤
+1. 构造 CDCClient，传入 CDCClientConfiguration
+2. 调用 CDCClient.connect，这一步是和 CDC Server 建立连接
+3. 调用 CDCClient.login，使用 server.yaml 中配置好的用户名和密码登录
+4. 调用 CDCClient.startStreaming，开启订阅，需要保证订阅的库和表在 ShardingSphere-Proxy 存在，否则会报错。
+
+> CDCClient.await 是阻塞主线程，非必需的步骤，用其他方式也可以，只要保证 CDC 线程一直在工作就行。
+
+如果需要更复杂数据消费的实现，例如写入到数据库，可以参考 [DataSourceRecordConsumer](https://github.com/apache/shardingsphere/blob/master/test/e2e/operation/pipeline/src/test/java/org/apache/shardingsphere/test/e2e/data/pipeline/cases/cdc/DataSourceRecordConsumer.java)
+
+#### 查看 CDC 任务运行情况
+
+CDC 任务的启动和停止目前只能通过 CDC Client 控制，可以通过在 proxy 中执行 DistSQL 查看 CDC 任务状态
+
+1. 查看 CDC 任务列表
+
+SHOW STREAMING LIST;
+
+运行结果
+
+```
+sharding_db=> SHOW STREAMING LIST;
+                     id                     |  database   | tables  | job_item_count | active |     create_time     | stop_time 
+--------------------------------------------+-------------+---------+----------------+--------+---------------------+-----------
+ j0302p0000702a83116fcee83f70419ca5e2993791 | sharding_db | t_order | 1              | true   | 2023-10-27 22:01:27 | 
+(1 row)
+```
+
+2. 查看 CDC 任务详情
+
+SHOW STREAMING STATUS j0302p0000702a83116fcee83f70419ca5e2993791;
+
+运行结果
+
+```
+sharding_db=> SHOW STREAMING STATUS j0302p0000702a83116fcee83f70419ca5e2993791;
+ item | data_source |          status          | active | processed_records_count | inventory_finished_percentage | incremental_idle_seconds | error_message
+------+-------------+--------------------------+--------+-------------------------+-------------------------------+--------------------------+---------------
+ 0    | ds_0        | EXECUTE_INCREMENTAL_TASK | true   | 1                       | 100                           | 101                      |
+ 1    | ds_1        | EXECUTE_INCREMENTAL_TASK | true   | 2                       | 100                           | 100                      |
+(2 rows)
+```
+
+3. 删除 CDC 任务
+
+DROP STREAMING j0302p0000702a83116fcee83f70419ca5e2993791;
+
+只有当 CDC 任务没有订阅的时候才可以删除，此时也会删除 openGauss 物理库上的 replication slots
+
+```
+sharding_db=> DROP STREAMING j0302p0000702a83116fcee83f70419ca5e2993791;
+SUCCESS
+```
+
+# 注意事项
+
+## 增量数据推送的说明
+
+1. CDC 增量推送目前是按照事务维度的，物理库的事务不会被拆分，所以如果一个事务中有多个表的数据变更，那么这些数据变更会被一起推送。
+如果要支持 XA 事务（目前只支持 openGauss），则 openGauss 和 Proxy 都需要 GLT 模块。
+2. 满足推送的条件是满足了一定大小的数据量或者到了一定的时间间隔（目前是 300ms），在处理 XA 事务时，收到的多个分库增量事件超过了 300ms，可能会导致 XA 事务被拆开推送。
+
+## 超大事务的处理
+
+目前是将大事务完整解析，这样可能会导致 CDC Server 进程 OOM，后续可能会考虑强制截断。
+
+## 建议的配置
+
+1. 如果有限流的要求，读写推荐只配置一侧，另一侧会自动受到限流的影响。
```

Comment:
Looks this suggestion is not valuable enough, new coming users could not understand it easily. Could we remove it?